tests/DistributedLocking/LockSet/DefaultLockSetTest.php

```diff
@@ -0,0 +1,85 @@
+<?php
+namespace Tests\DistributedLocking\LockSet;
```

Comment:
This came from looking at a couple of other projects (don't recall which) -- a nice benefit is that it means you can trivially exclude the test classes from a production-ready autoloader, which has some (small) advantages in terms of speed and autoloader size (if you generate a classmap instead of a dynamic autoloader, anyway).

I think it is a little cleaner too -- I'm pretty sure I like them separate, but could be convinced to change with sufficient debate.